routes/chorals.js

```diff
@@ -249,5 +275,39 @@ router.post('/edit/:choralId', function(req, res, next) {
     });
   });
 });
-          
+
+function getDefaultFuncs() {
+  // read all default choral funcs from their files
+  // return a promise that resolves to an array of strings,
+  // which are the stringified funcs
+
+  return new Promise((resolve, reject) => {
+    fs.readdir(DEFAULT_FUNCS_DIR, (err, fileNames) => {
+      if (err) {
+        reject(err);
+      }
+      else {
+        let fileContentPromises = fileNames
+          .map((fileName) => readFile(DEFAULT_FUNCS_DIR + '/', fileName));
+
+        // when all files are successfully read, return their contents
+        Promise.all(fileContentPromises)
```

Comment:
We should move the functions from the weather folder into the top level folder if we are going to do this. We cant see them as is.